backend/open_webui/routers/tasks.py

```
@@ -230,15 +250,16 @@ async def generate_title(
         raise e
 
     try:
-        return await generate_chat_completion(request, form_data=payload, user=user)
+        chat_title = await generate_chat_completion(request, form_data=payload, user=user)
+        ChatState.update(form_data["chat_id"], title_generated=True, chat_title=chat_title)
+        return chat_title
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)